### PR TITLE
Feature/add api calls

### PR DESCRIPTION
**Pull request by emilyjmiles**
#### What's this PR do?

- organizes component files
- add apiCalls and network requests

#### Where should the reviewer start?

- apiCalls file and App.js file

#### How should this be manually tested?

- pull down to local and make sure fetched data is working correctly

#### Any background context you want to provide?

- none

#### Are there any issues?

- not that I am aware of

#### Next steps:

- error handling

#### Questions:

 - none

### CODE REVIEW

File: src/Components/SingleMovie/SingleMovie.js [hunk 1]
```diff
@@ -2,19 +2,26 @@ import './SingleMovie.css';
 import backArrow from './backArrow.png';
 
 const SingleMovie = (props) => {
+    console.log(props.singleMovie.backdrop_path)
```
Comment (alyciacan) [inline src/Components/SingleMovie/SingleMovie.js:5]: Need to remove console log

File: src/Components/SingleMovie/SingleMovie.js [hunk 2]
```diff
@@ -2,19 +2,26 @@ import './SingleMovie.css';
...
+    const background = {
+        backgroundImage: `url(${props.singleMovie.backdrop_path})`,
+        backgroundSize: 'cover',
+        backgroundColor: 'rgba(0, 0, 0, 0.1)'
+    }
+
+    console.log(background)
```
Comment (alyciacan) [inline src/Components/SingleMovie/SingleMovie.js:12]: another console log

File: src/Components/SingleMovie/SingleMovie.js (hunk 1 above)
Comment (emilyjmiles) [reply src/Components/SingleMovie/SingleMovie.js:5]: Removed! Thank you!

File: src/Components/SingleMovie/SingleMovie.js (hunk 2 above)
Comment (emilyjmiles) [reply src/Components/SingleMovie/SingleMovie.js:12]: Removed! Thank you!

**Review by alyciacan — APPROVED** [review]
Thanks! Looks great!

**Pull request merged**